rubin_sim/moving_objects/make_lsst_obs.py

```diff
@@ -42,6 +43,12 @@ def setup_args(parser=None):
         "See https://github.com/oorb/oorb/tree/master/python#defining-orbits for "
         "additional documentation on the orbit file format. Default None.",
     )
+    parser.add_argument(
+        "--positions_file",
+        type=str,
+        default=None,
+        help="file with pre-computed positions for the objects in the orbit file",
+    )
```

Comment:
Could you please match the "help" string with the others in this file? 
"File with pre-computed 'rough' ephemerides for the objects in the orbit file. Default None, which will trigger computation of these rough ephemerides on-the-fly."